EvidenceApi/V1/Infrastructure/StringHasher.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+using EvidenceApi.V1.Infrastructure.Interfaces;
+
+namespace EvidenceApi.V1.Infrastructure
+{
+    public class StringHasher : IStringHasher
+    {
+        private readonly HashAlgorithm _hashAlgorithm;
+
+        public StringHasher()
+        {
+            _hashAlgorithm = SHA512.Create();
+        }
+
+        public string create(string toHash)
```

Comment:
```suggestion
        public string Create(string toHash)
```

Reply:
I need to get into the habit of using _PascalCase_